### PR TITLE
Add icons for Command Explorer

### PR DESCRIPTION
**Pull request by corbob**
Change refrech icon to use octicons (https://octicons.github.com)

## PR Summary

Add some icons for usability.
Add configuration to toggle visibility of Command Explorer
Needing to right click isn't super intuitive.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- NA PR has tests
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -456,6 +477,11 @@
       "type": "object",
       "title": "PowerShell Configuration",
       "properties": {
+        "powershell.sideBar.CommandExplorerVisibility": {
```
Comment (TylerLeonhardt) [inline package.json]: It looks like the `sideBar` is actually called the "Activity Bar". Can you change this to `activityBar`?

Comment (TylerLeonhardt) [reply package.json]:
```suggestion
        "powershell.activityBar.CommandExplorerVisibility": {
```

File: package.json [hunk 2]
```diff
@@ -456,6 +477,11 @@
...
+          "type": "boolean",
+          "default":true,
+          "description": "Specifies the visibility of the Command Explorer in the PowerShell sidebar."
```
Comment (TylerLeonhardt) [inline package.json]:
```suggestion
          "description": "Specifies the visibility of the Command Explorer in the PowerShell entry of the Activity Bar."
```

File: package.json [hunk 3]
```diff
@@ -202,6 +211,16 @@
...
     ],
     "menus": {
+      "commandPalette": [
+        {
+          "command": "PowerShell.InsertCommand",
+          "when": "false"
+        },
+        {
+          "command": "PowerShell.RefreshCommandsExplorer",
+          "when": "config.powershell.sideBar.CommandExplorerVisibility"
```
Comment (TylerLeonhardt) [inline package.json]:
```suggestion
          "when": "config.powershell.activityBar.CommandExplorerVisibility"
```

File: package.json [hunk 4]
```diff
@@ -72,7 +72,8 @@
       "PowerShellCommandExplorer": [
         {
           "id": "PowerShellCommands",
-          "name": "PowerShell Commands"
+          "name": "PowerShell Commands",
+          "when": "config.powershell.sideBar.CommandExplorerVisibility"
```
Comment (TylerLeonhardt) [inline package.json]:
```suggestion
          "when": "config.powershell.activityBar.CommandExplorerVisibility"
```

**Comment by corbob** [comment]
@TylerLeonhardt 

From my investigation, if you navigate to View > Appearance there's two options. There's `Toggle Side Bar` and `Hide Activity Bar`. If you select `Toggle Side Bar` it will toggle the view of the green highlighted section. While if you select "Hide Activity Bar" it will toggle the red highlighted section. Based on this behaviour, do you still want to change it to `activityBar` instead of `sideBar`? If we don't change it from `sideBar`, then the description at least should change to be `side bar`  to match the UI.

Reference screenshot:
image

**Comment by TylerLeonhardt** [comment]
Given this info. Ignore all my comments. :)

File: package.json [hunk 5]
```diff
@@ -456,6 +477,11 @@
       "type": "object",
       "title": "PowerShell Configuration",
       "properties": {
+        "powershell.sideBar.CommandExplorerVisibility": {
+          "type": "boolean",
+          "default":true,
```
Comment (bergmeister) [inline package.json]: There should've been a space after the colon (`:`). I opened PR #1688 to fix it. Running `Invoke-Build` at the end helps with that.

**Pull request merged**